test/test_qml.cpp

```diff
@@ -25,9 +25,30 @@
 #include <QQmlEngine>
 #include <QQmlFileSelector>
 #include <QtQuickTest>
+#include <qgis.h>
+#include <qgscoordinatereferencesystem.h>
 #include <qgsfeature.h>
 #include <qgsgeometry.h>
+#include <qgslocatormodel.h>
+#include <qgsmaplayer.h>
+#include <qgsmaplayerproxymodel.h>
+#include <qgsmapthemecollection.h>
 #include <qgspoint.h>
+#include <qgsproject.h>
+#include <qgsprojectdisplaysettings.h>
+#include <qgsquickcoordinatetransformer.h>
+#include <qgsquickelevationprofilecanvas.h>
+#include <qgsquickmapcanvasmap.h>
+#include <qgsquickmapsettings.h>
+#include <qgsquickmaptransform.h>
+#include <qgsrelationmanager.h>
+#include <qgssnappingutils.h>
+#include <qgsunittypes.h>
+#include <qgsvectorlayer.h>
+#include <qgsvectorlayereditbuffer.h>
+#include <qgswkbtypes.h>
+
+#include <valuemapmodel.h>
```

Comment:
one last thing @mohsenD98, this is one of our own header, it should go  up alongside the #include "positioning.h" and not use <>.